### ISSUE TITLE
Webhooks Details missing

### ISSUE THREAD

**Issue by Al-TSN** [gophish/gophish#1735]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 0.9

Brief description of the issue: The JSON returned for the Webhook has a Detail key that is blank. Would it make sense to return at least the campaign_id in that key or a separate key.  Currently there is no way to correlate the webhook, (other then the clicked message backtracking with the r_id) with the campaign.

What are you expecting to see happen? :  Campaign ID returned with the webhook

What are you seeing happen?  There is not enough detailed information to correlate the webhook data

**Comment by jordan-wright** [comment]
Hey @Al-TSN!

Great catch. I should have thought about that earlier. We do have access to that field, but historically haven't included it in the JSON payload. I can get a fix in for that ASAP.

**Comment by jordan-wright** [comment]
Hey @Al-TSN,

I managed to get this fixed in 3d525525ea4ff9fc99d42b077653b09e4e6a43d8.

Here's what the new payload looks like:

```json
POST / HTTP/1.1
Host: localhost:9999
Accept-Encoding: gzip
Content-Length: 130
Content-Type: application/json
User-Agent: Go-http-client/1.1
X-Gophish-Signature: sha256=1915f5edce29d7aa3af83a9f639bf174d7150d1b7dc47e405b4225e02d061563
{
    "campaign_id": 5,
    "email": "Jennifer.Todd.661@example.com",
    "time": "2020-01-30T13:13:10.518322Z",
    "message": "Email Sent",
    "details": ""
}
```

Thanks again for sending this over!
